modules/installation-ibm-cloud-regions.adoc

```diff
@@ -19,3 +30,42 @@ You can deploy an {product-title} cluster to the following regions:
 * `jp-tok` (Tokyo, Japan)
 * `us-east` (Washington DC, United States)
 * `us-south` (Dallas, United States)
+endif::ibm-power-vs[]
+ifdef::ibm-power-vs[]
+//Not listed for openshift-install: br-sao, in-che, kr-seo
+
+* `dal` (Dallas, USA)
+** `dal12`
+* `us-east` (Washington DC, USA)
+** `us-east`
+* `eu-de` (Frankfurt, Germany)
+** `eu-de-1`
+** `eu-de-2`
+* `lon` (London, UK)
+** `lon04`
+** `lon06`
+* `osa` (Osaka, Japan)
+** `osa21`
+* `sao` (Sao Paulo, Brazil)
+** `sao21`
```

Comment:
zone is named 'sao01' (not sao21). Please correct it. @alishaIBM

Reply:
Acknowledge.